airbyte-integrations/connectors/source-insightly/unit_tests/test_incremental_streams.py

```diff
@@ -23,25 +22,40 @@ def patch_incremental_base_class(mocker):
 
 def test_cursor_field(patch_incremental_base_class):
     stream = IncrementalInsightlyStream(authenticator=authenticator, start_date=start_date)
-    # TODO: replace this with your expected cursor field
     expected_cursor_field = "DATE_UPDATED_UTC"
     assert stream.cursor_field == expected_cursor_field
 
 
+def test_incremental_params(patch_incremental_base_class):
+    stream = IncrementalInsightlyStream(authenticator=authenticator, start_date=start_date)
```

Comment:
@henriblancke Can you add a docstring explaining the reason of 1 secod subtract. This will help for future users understand the code.
* There is any docs to Insightly which says that? If not add: talked to Insightly teams...

Reply:
Unfortunately no docs, we had to reach out to their dev team